### ISSUE TITLE
Somewhat off-topic with hackintosh but I would like to know if hackintoshing might cause an issue with pop os booting in uefi

### ISSUE THREAD

**Issue by aaronjrodrigues** [wmchris/DellXPS15-9550-OSX#176]
I am trying to install pop os on my dell xps 15 9550, using uefi (this will be important later) but it shows the live boot selection screen but after selecting live boot it shows a black screen. However it will boot in legacy unfortunately I cannot use legacy boot to install (due to an error because of the way my disk is partitioned).

The funny thing is that ubuntu boots fine but not pop os even though they are both based on ubuntu.

So basically I wanted to ask if anyone who has macos installed if they could check if pop os boots into the live enviroment (in uefi) for their xps 15, I wanted to check if hackintoshing might cause pop os to not boot and this felt like the right place to ask, I am very sorry that this is kinda off topic with hackintoshing.

**Comment by ghost** [comment]
@aaronjrodrigues I have had experience similar issue when I tried Pop OS 6 months ago. The issue was the tool used to create the live USB. The only one that worked was netbootin. Even the command DD will not make the live USB boot. I spent hours trying to figure out. But it was that. I tried Rufus, dd, and some other ones. Try unetbootin.
